src/main/java/net/glowstone/GlowWorld.java

```diff
@@ -1,5 +1,6 @@
 package net.glowstone;
 
+import io.netty.util.internal.ConcurrentSet;
```

Comment:
Please consider using another implementation. Netty uses this API only internal and they are not part of the API.